### ISSUE TITLE
[RenderTargets] Auto options for depth/stencil result in black screen on android.

### ISSUE THREAD

**Issue by Disar** [Kode/Kha#661]
I have tested two devices with two different gpu sets:

- Mali-T720MP2
- Adreno 305

Both give a black screen.

Code:

`backbuffer = Image.createRenderTarget(360,640,null,kha.graphics4.DepthStencilFormat.DepthAutoStencilAuto);`

**Comment by Disar** [comment]
Further testing shows that Depth16 also draws properly with g4. Might be gpu specific. It works on the Mali gpu.

**Comment by RobDangerous** [comment]
Yes, that kinda thing is GPU specific, but auto should always work...
